js/main.js

```diff
@@ -5,14 +5,17 @@ import {createAllMarkers} from './map.js';
 import {showAlert} from './message.js';
 import {getData} from './api.js';
 import {setOnFiltersChange} from './filter-user.js';
+import { turnOnMapFilters, turnOffMapFilters} from './filter.js';
 
 
-const ADS_NUM = 10;
+const ADS_LIMIT = 10;
 
 getData(
   (ads) => {
-    createAllMarkers(ads.slice(0, ADS_NUM));
-    setOnFiltersChange(createAllMarkers, ads);
+    turnOnMapFilters();
+    createAllMarkers(ads.slice(0, ADS_LIMIT));
+    setOnFiltersChange({ createAllMarkers, removeAllMarkers }, data);
```

Comment:
```suggestion
    setOnFiltersChange({ createAllMarkers, removeAllMarkers }, ads);
```
И `removeAllMarkers` нужно импортировать из `./map.js`.